PetaPoco/Database.cs

```diff
@@ -3365,7 +3373,9 @@ public string FormatCommand(string sql, object[] args)
                 sb.Append("\n");
                 for (int i = 0; i < args.Length; i++)
                 {
-                    sb.AppendFormat("\t -> {0}{1} [{2}] = \"{3}\"\n", _paramPrefix, i, args[i].GetType().Name, args[i]);
+                    var arg = args[i];
+                    var argTypeName = arg == null ? "Unknown Type" : args.GetType().Name;
+                    sb.AppendFormat("\t -> {0}{1} [{2}] = \"{3}\"\n", _paramPrefix, i, argTypeName, arg);
                 }
```

Comment:
This should have the same result, while remaining as concise as possible:

```suggestion
                    sb.AppendFormat("\t -> {0}{1} [{2}] = \"{3}\"\n", _paramPrefix, i, args[i]?.GetType().Name ?? "Unknown Type", args[i]);
```